INSTALL.rst

```diff
@@ -257,6 +257,27 @@ location specified in ``haas.cfg``) and initialize its tables::
 
   haas init_db
 
+
+Authentication and Authorization
+--------------------------------
+
+HaaS includes a pluggible architecture for authentication and authorization.
+Right now, the only authentication backend uses HTTP basic auth, with usernames
+and password stored in the database. to enable this, Make sure the
```

Comment:
Could you make it more clear that not including an extension will cause auth* to be disabled?

Also, please fix the capitalization on this line.